bin/ingest_local

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
This is seemingly inconsistent with Ingester PSC, DK, OC post-refactor, but I'm not sure what the best name is. If it's true that there's no local version of the ingesters present in those projects, then I think this should probably be named `ingest-local`, to be similar to Transformer PSC `transform-local` and my other comments in this review.